proto/apidocs.swagger.json

```diff
@@ -103,23 +103,7 @@
             "in": "body",
             "required": true,
             "schema": {
-              "type": "object",
-              "properties": {
-                "metadata": {
-                  "type": "object",
-                  "description": "Target room metadata."
-                },
-                "status": {
-                  "type": "string",
-                  "description": "Indicates the room status."
-                },
-                "timestamp": {
-                  "type": "string",
-                  "format": "int64",
-                  "description": "Timestamp of the ping event."
-                }
-              },
-              "description": "The ping request."
+              "$ref": "#/definitions/v1UpdateRoomWithPingRequest"
```

Comment:
@mflilian I guess this file change should not be included in this PR, right?

Reply:
YESS!!!!!!

Reply:
I'm gonna remove it, thanks Gui